packages/frontend/vite.config.ts

```diff
@@ -1,33 +1,21 @@
-import { nodeTypes } from "@mdx-js/mdx";
+import mdx from "@mdx-js/rollup";
 import rehypeKatex from "rehype-katex";
-import rehypeRaw from "rehype-raw";
 import remarkMath from "remark-math";
 import { defineConfig } from "vite";
 import solid from "vite-plugin-solid";
 import wasm from "vite-plugin-wasm";
 
-// @ts-expect-error Types are missing.
-// *Also*, this plugin causes Vite 5 to complain about CJS.
-// https://github.com/nksaraf/vinxi/issues/289
-import pkg from "@vinxi/plugin-mdx";
-const { default: mdx } = pkg;
-
 export default defineConfig({
     plugins: [
         wasm(),
-        mdx.withImports({})({
-            jsx: true,
-            jsxImportSource: "solid-js",
-            providerImportSource: "solid-mdx",
-            rehypePlugins: [
-                [rehypeRaw, { passThrough: nodeTypes }],
-                [rehypeKatex, { throwOnError: false }],
-            ],
-            remarkPlugins: [[remarkMath, { singleDollarTextMath: true }]],
-        }),
-        solid({
-            extensions: [".mdx", ".md"],
```

Comment:
Removed because it is superfluous:
- we don't have any `.md` file and I'm pretty sure they wouldn't work correctly anyway.
- `.mdx file` are handled by `@mdx-js/rollup`. The docs for  `vite-plugin-solid` mention adding `.mdx`, but I can't figure out what the usecase for that actually is.

Reply:
Sounds good. As I noted below, I stumbled my way through this without understanding how all the pieces fit together.